src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java

```diff
@@ -64,5 +65,7 @@
 
   List<byte[]> aclCat(byte[] category);
 
+  String aclLog(byte[] options);
```

Comment:
`aclLog()` and `aclLog(int limit)` can also be added here.

Reply:
Since I have chosen to return a `List<AccessControlLogEntry>` I have not added these methods here.

If I return `byte[]` I can add them, but like on previous comment it is not clear to me what is the good practice for binary API

Reply:
`List<AccessControlLogEntry>` should be fine in BinaryJedis as well. If there is user demand, we can update `AccessControlLogEntry` to serve `byte[]` too. For example, `Tuple` is used in both binary and string codes.

Reply:
I have added 

```
  List<byte[]> aclLogBinary();

  List<byte[]> aclLogBinary(int limit);
```
Returning a simple list of `byte[]`, in case developer wants to use the raw data.